packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/trees/TreeControllerIntegrationTests.java

```diff
@@ -177,38 +200,42 @@ public void givenProjectWhenWeSubscribeToTreeEventsOfItsExplorerThenTheTreeOfThe
                 .verify(Duration.ofSeconds(10));
     }
 
-    @Test
+    private static Stream<Arguments> provideDeleteTreeItemThenAndExpected() {
```

Comment:
Why go through all this complexity for something like that? Is that more readable and maintainable?